apps/prairielearn/src/migrations/20241007203121_draft_questions.sql

```diff
@@ -0,0 +1,34 @@
+ALTER TABLE questions
+ADD COLUMN IF NOT EXISTS is_draft boolean NOT NULL DEFAULT (false);
+
+ALTER TABLE pl_courses
+ADD COLUMN IF NOT EXISTS draft_number integer NOT NULL DEFAULT (0);
+
+CREATE TABLE IF NOT EXISTS questions_draft_metadata (
+  id bigint NOT NULL PRIMARY KEY,
```

Comment:
Is this meant to refer to a question? If so, it should be named `question_id` and should have proper foreign key constraints.

As a rule, we always give tables a `bigserial` primary key, even if they're uniquely identified by other columns.

Reply:
Changed over in a94d9c043b56e640362369ff73f52d0a019803dd.